db/migrate/20220526132409_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :likes do |t|
```

Comment:
in the future when naming tables, it's best to be as specific as possible, like `recipe_likes` (but go ahead and leave it for this exercise)

Reply:
make sence, thanks for the feedback!